docs/contracts/instance-query-validation-breaking-changes.md

```
@@ -0,0 +1,334 @@
+# Now: 400 — Validation:900012 / sort.invalidJson
+curl -i "http://localhost:4201/api/v1/kredi/workflows/basvuru/instances?sort=-createdAt"
+curl -i "http://localhost:4201/api/v1/kredi/workflows/basvuru/instances?sort=createdAt"
+curl -i "http://localhost:4201/api/v1/kredi/workflows/basvuru/instances?orderBy=-modifiedAt"
+```
+
+`sort` has always required JSON. A bare field name is not valid JSON, so it was never parsed — the
+shorthand only ever appeared to work because the failure was swallowed.
+
+**Fix:** use the JSON form.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)